### ISSUE TITLE
[Extractor] encoding in policy.xml Files

### ISSUE THREAD

**Issue by StevonixDev** [Azure/azure-api-management-devops-resource-kit#371]
Hi,
when extracting the policies into seperate XML files I have following issue. ( < becomes \&lt;)

```
                    <when condition="@(context.Response.StatusCode  < 500)">
                        <set-status code="404" />
                    </when>

```
in the portal is extracted to:

> 			<when condition="@(context.Response.StatusCode  &lt; 500)">
> 				<set-status code="404" />
> 			</when>

When using the ARM Templates this results in an error.

My config file:
{
  "sourceApimName": "apim",
  "destinationApimName": "apim2",
  "resourceGroup": "apimrg",
  "fileFolder": "C:\\apim-extractor\\ARM",
  "linkedTemplatesBaseUrl": "https://xxx.blob.core.windows.net/xxx",
  "policyXMLBaseUrl": "https://xxx.blob.core.windows.net/xxx/policies",
  "splitAPIs": "true",
  "paramServiceUrl": "true",
  "paramNamedValue": "true"
}

regards 
Stefan

**Comment by bremnes** [comment]
Hi @StevonixDev,

This seems like an issue we had back in the days, #230. 

I'm guessing this is a global policy? It seems like [that code](https://github.com/Azure/azure-api-management-devops-resource-kit/blob/master/src/APIM_ARMTemplate/apimtemplate/Extractor/EntityExtractors/PolicyExtractor.cs#L22) isn't fetched in the correct format - it's missing the `format=rawxml` which can be seen in the [APIExtractor](https://github.com/Azure/azure-api-management-devops-resource-kit/blob/master/src/APIM_ARMTemplate/apimtemplate/Extractor/EntityExtractors/APIExtractor.cs#L162).

If you can confirm it, it should be a quick fix :)

**Comment by StevonixDev** [comment]
Thank you for your response. This issue is causing a lot of pain for our developers. I don't want to quick fix it with a hack because we really appreciate the extractor tool and want to stay compatible to updates.

Your guess is correct!! It is a global policy.


I downloaded the latest  version of the extractor with source code you linked to:

            string requestUrl = string.Format("{0}/subscriptions/{1}/resourceGroups/{2}/providers/Microsoft.ApiManagement/service/{3}/apis/{4}/operations/{5}/policies/policy?api-version={6}&format=rawxml",
               baseUrl, azSubId, ResourceGroupName, ApiManagementName, ApiName, OperationId, GlobalConstants.APIVersion);

But still the I am getting an encoded policy.xml.




Did I understand you correct or must I set a parameter in the Configfile?

regards
Stefan
